Remove references to removed modules for compatibility with PEG.js HEAD.

r3c opened the pull request.

nylen commented:
This seems like a good thing to go ahead and get in.  From discussion at #6, let's also do the following:

- Update the `pegjs` version in `package.json` to a specific version (remove the `^`)
- Clearly indicate the intended compatible `pegjs` version in the readme

r3c commented:
Sure, will do. Thanks for your feedback :)

r3c committed:
Remove references to removed modules for compatibility with PEG.js HEAD.

nylen committed:
Update compatible PEG.js version

We can't guarantee future-compatibility so list only the known tested
release version of PEG.js.
